### PR TITLE
Misc markdown rules fixes for 7.5

### PR DESCRIPTION
**Pull request by ArieHein**
# PR Summary

* Majority are MD049 Emphasis. replacing *text* with _text_ per MD rules
* Few MD038 for extra spaces in code

Two left overs MD026 and MD31 from previous sessions.

2 more small PRs to go after this and 7.5 should be in great shape !

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

File: reference/7.5/Microsoft.PowerShell.Diagnostics/Get-WinEvent.md [hunk 1]
```diff
@@ -500,7 +500,7 @@ from newest to oldest.
 
 Event Tracing for Windows (ETW) writes events to the log as events occur. The events are stored in
 the order of oldest to newest. An archived ETW file is saved as an `.etl` such as **TraceLog.etl**.
-The events are listed in the order in which they are written to the log, so the *Oldest* parameter
+The events are listed in the order in which they are written to the log, so the _Oldest_ parameter
```
Comment (sdwheeler) [inline reference/7.5/Microsoft.PowerShell.Diagnostics/Get-WinEvent.md:503]: Parameter names (in prose) should be bold - see https://learn.microsoft.com/en-us/powershell/scripting/community/contributing/editorial-checklist?view=powershell-7.4#formatting
```suggestion
The events are listed in the order in which they are written to the log, so the **Oldest** parameter
```

File: reference/7.5/Microsoft.PowerShell.Management/Join-Path.md [hunk 2]
```diff
@@ -140,7 +140,7 @@ In this example, no parameter names are used, thus "a" binds to `Path`, "b" to `
 
 ### -AdditionalChildPath
 
-Specifies additional elements to append to the value of the *Path* parameter. The `ChildPath`
+Specifies additional elements to append to the value of the _Path_ parameter. The `ChildPath`
```
Comment (sdwheeler) [inline reference/7.5/Microsoft.PowerShell.Management/Join-Path.md:143]:
```suggestion
Specifies additional elements to append to the value of the **Path** parameter. The **ChildPath**
```

File: reference/7.5/Microsoft.PowerShell.Management/Set-Item.md [hunk 3]
```diff
@@ -171,7 +171,7 @@ Forces the cmdlet to set items that cannot otherwise be changed, such as read-on
 variables. The cmdlet cannot change constant aliases or variables.
 Implementation varies from provider to provider.
 For more information, see [about_Providers](../Microsoft.PowerShell.Core/About/about_Providers.md).
-Even using the *Force* parameter, the cmdlet cannot override security restrictions.
+Even using the _Force_ parameter, the cmdlet cannot override security restrictions.
```
Comment (sdwheeler) [inline reference/7.5/Microsoft.PowerShell.Management/Set-Item.md:174]:
```suggestion
Even using the **Force** parameter, the cmdlet cannot override security restrictions.
```

File: reference/7.5/Microsoft.PowerShell.Utility/Unblock-File.md [hunk 4]
```diff
@@ -70,7 +70,7 @@ PS C:\> dir C:\Downloads\*PowerShell* | Unblock-File
 
 This command shows how to find and unblock PowerShell scripts.
 
-The first command uses the **Stream** parameter of the *Get-Item* cmdlet get files with the
+The first command uses the **Stream** parameter of the _Get-Item_ cmdlet get files with the
```
Comment (sdwheeler) [inline reference/7.5/Microsoft.PowerShell.Utility/Unblock-File.md:73]: Cmdlet names should be in code spans per our style guide.
```suggestion
The first command uses the **Stream** parameter of the `Get-Item` cmdlet get files with the
```

**Review by sdwheeler — CHANGES_REQUESTED** [review]
See my comments and suggestions.

File: reference/7.5/Microsoft.PowerShell.Core/About/About.md [hunk 5]
```diff
@@ -59,7 +59,7 @@ PowerShell provides the ability to dynamically add new properties and alter the
...
+A generic method is a method with two parameter lists: a list of generic types and a list of method arguments.  The following examples show the new PowerShell syntax for accessing a generic method:  ```Syntax # static generic methods [type_name]::MethodName[generic_type_arguments](method_arguments)  # instance generic methods $object.MethodName[generic_type_arguments](method_arguments)```  The `generic_type_arguments` can be a single type or comma-separated list of types, like `[string, int]`, including other generic types like `$obj.MethodName[string, System.Collections.Generic.Dictionary[string, int]]()`  The `method_arguments` can be zero or more items.  For more information, see [Generics in .NET](/dotnet/standard/generics/).
```
[the remark this answers is not in the document]
Comment (sdwheeler) [reply reference/7.5/Microsoft.PowerShell.Core/About/About.md:62]: This is fixed now. You shouldn't need to include it in your PR. Please resync the main branch and rebase.

File: reference/7.5/Microsoft.PowerShell.Diagnostics/Get-WinEvent.md (hunk 1 above)
Comment (ArieHein) [reply reference/7.5/Microsoft.PowerShell.Diagnostics/Get-WinEvent.md]: Ok, so the original was wrong in this case. From MD perspective, it just pointed to the wrong character being used.

File: reference/7.5/Microsoft.PowerShell.Utility/Unblock-File.md (hunk 4 above)
Comment (ArieHein) [reply reference/7.5/Microsoft.PowerShell.Utility/Unblock-File.md]: Ok, so the original was wrong in this case. From MD perspective, it just pointed to the wrong character being used.

**Pull request closed without merging**